api/src/main/java/jakarta/data/repository/Pageable.java

```diff
@@ -54,20 +51,29 @@
  *     with the <code>First</code> keyword.</li>
  * </ul>
  */
-public class Pageable {
+public interface Pageable {
 
-    private static final long DEFAULT_SIZE = 10;
-
-    final long page;
-
-    final long size;
-
-    final List<Sort> sorts;
+    /**
+     * Creates a new <code>Pageable</code> with the given page number and with a default size of 10.
+     *
+     * @param pageNumber The page number.
+     * @return a new instance of <code>Pageable</code>.
+     * @throws IllegalArgumentException when the page number is negative or zero.
+     */
+    public static Pageable ofPage(long pageNumber) {
```

Comment:
clarify that the return value is never null

Reply:
I update this occurrence and all of the similar places in commit https://github.com/jakartaee/data/pull/68/commits/b636074d94404592644014b165acd07a2827bafc along with the update to add `final`.  I haven't done anything yet about the record idea, but I think we should, but wanted to give it some more thought first.

Reply:
I guess  "a new instance of <code>Pageable</code>." it is enough.
I mean, how a new instance can be null? 
IMHO it is redundant.

Reply:
> I guess "a new instance of `Pageable`." it is enough. I mean, how a new instance can be null? IMHO it is redundant.

It seemed redundant to me as well, but I didn't see any harm including the additional statement in the javadoc, so I agreed to do so.  @otaviojava did you see any other issues with this during your review, or are we okay to merge?

Reply:
IMHO @njr-11 merge it!